### PR TITLE
docs: 2024-11-22 changelog

### PR DESCRIPTION
**Pull request by danieltprice**
https://neon-next-git-dprice-2024-11-22-changelog-neondatabase.vercel.app/docs/changelog/2024-11-22

### CODE REVIEW

File: content/changelog/2024-11-22.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+<summary>**Fixes & improvements**</summary>
+
+- **IP Allow**
+
+  We addressed an issue for IP Allow users connecting over VPN where an **Access Denied** modal appeared repeatedly on the **SQL Editor** and **Tables** pages in the Neon Console. To prevent this, we added a "Do not ask again" checkbox to allow users to silence the modal.
+
+- **Time Travel Assist**
+
+  Ephemeral compute timeouts for [Time Travel Assist](/docs/guides/time-travel-assist) have been increased from 10 to 30 seconds. Time Travel Assist enables querying any point in your history using temporary branches and computes, which are automatically cleaned up after use. After 30 seconds of inactivity, the branch is deleted, and the endpoint is removed.
```
Comment (ololobus) [inline content/changelog/2024-11-22.md:47]:
```suggestion
  Ephemeral compute suspend timeouts for [Time Travel Assist](/docs/guides/time-travel-assist) have been increased from 10 to 30 seconds. Time Travel Assist enables querying any point in your history using temporary branches and computes, which are automatically cleaned up after use. After 30 seconds of inactivity, the branch is deleted, and the endpoint is removed.
```

File: content/docs/guides/time-travel-assist.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ However, you can see the history of operations related to the creation and delet
 
 ### How long do ephemeral endpoints remain active
 
-The ephemeral endpoints are created according to your configured [default compute size](/docs/manage/projects#reset-the-default-compute-size). An ephemeral compute remains active for as long as you keep running queries against it. After 10 seconds of inactivity, the timeline is deleted and the endpoint is removed.
+The ephemeral endpoints are created according to your configured [default compute size](/docs/manage/projects#reset-the-default-compute-size). An ephemeral compute remains active for as long as you keep running queries against it. After 30 seconds of inactivity, the timeline is deleted and the endpoint is removed.
```
Comment (ololobus) [inline content/docs/guides/time-travel-assist.md:29]: Random thought: it's interesting that we mention them here, but we don't show them in the docs

**Review by ololobus — APPROVED** [review]
Ephemeral and extensions part LGTM

File: content/docs/guides/time-travel-assist.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/guides/time-travel-assist.md:29]: @ololobus We have it here: https://neon.tech/docs/guides/time-travel-assist#how-long-do-ephemeral-endpoints-remain-active
Maybe it's too buried.

**Comment by clarkbw** [comment]
The extensions page doesn't show `timescaledb` under v17
https://neon-next-git-dprice-2024-11-22-changelog-neondatabase.vercel.app/docs/extensions/pg-extensions

**Comment by danieltprice** [comment]
> The extensions page doesn't show `timescaledb` under v17 https://neon-next-git-dprice-2024-11-22-changelog-neondatabase.vercel.app/docs/extensions/pg-extensions
Got it. Thank you.
image

**Pull request merged**